### ISSUE TITLE
ContainerFactory hardcodes some directories to /var/…

### ISSUE THREAD

**Issue by fritzmg** [contao/installation-bundle#34]
Try the following scenario:

1. Install Contao `4.2.5` via `composer create-project contao/standard-edition <target> 4.2.5`.
2. Open `example.org/install.php`.
3. Update the installation to Contao `4.3.0` via `composer update`.
4. Delete the `/system/logs` symlink.
5. Open `example.org/install.php`.

The creation of the symlink for `/system/logs` in the Contao Install Tool will fail, because the `ContainerFactory` hardcodes the log directory for the kernel to `/var/logs`: [ContainerFactory.php#L53](https://github.com/contao/installation-bundle/blob/1.2.0/src/DependencyInjection/ContainerFactory.php#L53). This directory will only be present, if you directly installed the `contao/standard-edition 4.3`, instead of updating to it.

Doing `php app/console contao:symlinks` instead of using the Install Tool will work.

\cc @stefansl

**Comment by fritzmg** [comment]
Another problem occurs when the `InstallationController` generates the `bootstrap.php.cache` file [here](https://github.com/contao/installation-bundle/blob/1.2.0/src/Controller/InstallationController.php#L152). The cache directory is hardcoded to `/var/cache`: [ContainerFactory.php#L52](https://github.com/contao/installation-bundle/blob/1.2.0/src/DependencyInjection/ContainerFactory.php#L52) - thus the new `bootstrap.php.cache` file will be generated in `/var` instead.

And if you try to go to `example.org/contao/install` afterwards, you'll see the following error message multiple times:
```
Warning: file_get_contents(/var/www/contao4test/htdocs/app/bootstrap.php.cache): failed to open stream:
No such file or directory in /var/www/contao4test/htdocs/vendor/symfony/symfony/src/Symfony/Component/ClassLoader/ClassCollectionLoader.php on line 153
```
Which is weird, because the _old_ `bootstrap.php.cache` definitely still exists in that location.

Reproduction (basically the same as above):

1. Install Contao 4.2.5 via `composer create-project contao/standard-edition <target> 4.2.5`.
2. Open `example.org/install.php`.
3. Update the installation to Contao 4.3.0 via composer update.
4. Open `example.org/install.php`.

The aforementioned Warnings will be shown.

Running
```
composer run-script post-update-cmd
```
does __not__ fix it. If you delete `/app/cache/*`, the following errors will be shown:
```
Warning: require(/var/www/contao4test/htdocs/app/../../../../../vendor/symfony/symfony/src/Symfony/Component/ClassLoader/ClassCollectionLoader.php): failed to open stream: No such file or directory in /var/www/contao4test/htdocs/app/bootstrap.php.cache on line 4

Fatal error: require(): Failed opening required '/var/www/contao4test/htdocs/app/../../../../../vendor/symfony/symfony/src/Symfony/Component/ClassLoader/ClassCollectionLoader.php' (include_path='.:/usr/share/php:/usr/share/pear') in /var/www/contao4test/htdocs/app/bootstrap.php.cache on line 4
```

**Comment by stefansl** [comment]
I can confirm this. Tested this on two different linux machines.

**Comment by leofeyer** [comment]
We need to use `$cacheDir = $kernel->getCacheDir()` and `$logsDir = $kernel->getLogsDir()`.

**Comment by leofeyer** [comment]
Fixed in 53370d95669ff5084ab6ef64bdcae699213c6f6d.
